Configuration/NodeTypes.Finishers.Confirmation.yaml

```diff
@@ -21,6 +21,11 @@
           # Alternatively enable a rich text editor:
           # editor: 'Neos.Neos/Inspector/Editors/RichTextEditor'
           # editorOptions:
+          #   excludeNodeTypes:
+          #     'Neos.Form.Builder:ElementCollection':
+          #       exclude: true # excludes only the element
+          #       excludeChildren: false # includes all child-elements
```

Comment:
"children" is the wrong notion here. A *node* has child nodes, a node *type* has *super* types.
I would suggest to simplify the syntax to a single list of `true/false` values like we do it in other places (e.g. node type constraints):

```yaml
excludeNodeTypes:
  'Neos.Form.Builder:ElementCollection': true
  'Some.Specific:ElementCollection': false
```

And, to make it even more readable, we could swap the "deny list" to an "allow list", as in:

```yaml
nodeTypes:
  'Neos.Form.Builder:ElementCollection': false
  'Some.Specific:ElementCollection': false
```

Reply:
I think only to have the simple syntax is not a good solution because my solution allows your objection and an easier configuration.

A boolean value allows to hide or show the entire element and its child nodes.
```yaml
nodeTypes:
  'Neos.Form.Builder:ElementCollection': false
```
(based on your objection this would exclude a node of this node type)

Or with the more specific syntax, the entire element can be hidden, but not his child nodes or just the child nodes. 
```yaml
nodeTypes:
  # hides the node type, shows the child nodes
  'Neos.Form.Builder:ElementCollection':
    excludeNodeType: true
    excludeChildNodes: false

  # shows the node type, hides the child nodes
  'Neos.Form.Builder:ElementCollection':
    excludeNodeType: false
    excludeChildNodes: show
```

The complex syntax is helpful for the validators or the `SelectOptionCollection`, for example, since only the validator collection has to be hidden here and not every single validator.


Thats the configuration I'm using currently in my project (maybe that helps more to understand why I solved it that way):
```yaml
excludeNodeTypes:
  'Foo.Bar.FormBuilder:Grid': &includeOnlyChildren
    exclude: true
    excludeChildren: false
  'Foo.Bar.FormBuilder:Column': *includeOnlyChildren
  'Foo.Bar.FormBuilder:NavigationNext': false
  'Foo.Bar.FormBuilder:NavigationPrevious': false
  'Foo.Bar.FormBuilder:Image': true
  'DL.HoneypotFormField:HoneypotField': true
  'Neos.Form.Builder:StaticText': true
  'Neos.Form.Builder:ValidatorCollection': true
  'Neos.Form.Builder:ElementCollection': *includeOnlyChildren
  'Neos.Form.Builder:SelectOptionCollection': *includeOnlyChildren
```

In the new syntax it would look like this:
```yaml
nodeTypes:
  'Foo.Bar.FormBuilder:Grid': &includeOnlyChildNodes
    excludeNodeType: true
    excludeChildNodes: false
  'Foo.Bar.FormBuilder:Column': *includeOnlyChildNodes
  'Foo.Bar.FormBuilder:NavigationNext': false
  'Foo.Bar.FormBuilder:NavigationPrevious': false
  'Foo.Bar.FormBuilder:Image': true
  'DL.HoneypotFormField:HoneypotField': true
  'Neos.Form.Builder:StaticText': true
  'Neos.Form.Builder:ValidatorCollection': true
  'Neos.Form.Builder:ElementCollection': *includeOnlyChildNodes
  'Neos.Form.Builder:SelectOptionCollection': *includeOnlyChildNodes
```